bootstrap/peer.go

```diff
@@ -76,6 +77,11 @@ type Peer struct {
 		Inspector    *kademlia.Inspector
 	}
 
+	Contact struct {
```

Comment:
do we need a comment here?

Reply:
Not according to the linter, and not according to current practice (see the other nearby structs). But we could sure add one if it is needed for readability

Reply:
no need to add one, i just thought the linter would complain